lib/helper.coffee

```diff
@@ -0,0 +1,91 @@
+ChildProcess = require 'child_process'
+Path = require 'path'
+FS = require 'fs'
+{XRegExp} = require 'xregexp'
+
+class Helpers
+  @findFile: (directory, fileName) ->
```

Comment:
keeping it as a class allows you to fat-arrow the functions so they can call each other when needed.